### ISSUE TITLE
Samples on reverberation set

### ISSUE THREAD

**Issue by xxoospring** [huyanxin/DeepComplexCRN#6]
Model seems only remove noise on samples but not reverberation, is that right? If the model can remove both noise and reverb?

**Comment by huyanxin** [comment]
No, when training dccrn, the targets are reverberant clean speech. If you want to remove both noise and reverb, using early-reverb clean speech like other teams may be better.
